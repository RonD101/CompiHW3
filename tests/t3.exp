---end scope---
i1 INT 0
i2 INT 1
b1 BYTE 2
b2 BYTE 3
bo1 BOOL 4
bo2 BOOL 5
ires INT 6
bres BYTE 7
bores BOOL 8
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
main ()->VOID 0
