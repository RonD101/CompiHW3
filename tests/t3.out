1
2
3
4
5
6
7
9
10
11
13
13
13
14
14
14
16
16
16
17
17
17
19
19
19
20
20
20
21
21
23
23
23
24
24
24
25
25
25
26
26
26
27
27
27
28
28
28
29
29
29
30
30
30
32
32
32
33
33
33
34
34
34
35
35
35
36
36
36
37
37
37
38
38
38
39
39
39
---end scope---
i1 INT 0
i2 INT 1
b1 BYTE 2
b2 BYTE 3
bo1 BOOL 4
bo2 BOOL 5
ires INT 6
bres BYTE 7
bores BOOL 8
---end scope---
print (STRING)->VOID 0
printi (INT)->VOID 0
main ()->VOID 0
